src/commands/consolidate.py

```diff
@@ -196,8 +197,8 @@ def consolidate(
 
 async def main(
     vault_address: ChecksumAddress,
-    source_public_keys: list[HexStr],
-    target_public_key: HexStr,
+    source_public_keys: list[HexStr] | None,
+    target_public_key: HexStr | None,
```

Comment:
(optional)
At this point source and  target keys are both None or both not None. Suggestion: pack them in dataclass:
```
class ConsolidationKeys:
    source_keys: list[HexStr]
    target_key: HexStr
```
Then
```suggestion
    consolidation_keys: ConsolidationKeys | None
```